content/app/github/commit-page-helper.js

```diff
@@ -0,0 +1,53 @@
+define("content/app/github/commit-page-helper", function(require, exports, module) {
+
+	// imports
+	var FileElementWrapper = require("content/app/github/wrappers/dom-element-wrapper");
+
+	/**
+	 * Helper for accessing and manipulating DOM of commit and pull request page.
+	 */
+	var CommitPageHelper = Class(Object, {
+
+		$statics : {
+
+			FILE_CLASS : 'file',
```

Comment:
I think that those classes which are controlled by git should be stored in some configurable JSON. That would give us extremely easy way to maintain functionality if GitHub changes layout, etc.